CHANGELOG.md

```diff
@@ -22,41 +22,23 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased
+## 1.3.0 - 2020-05-21
 * ### ALL
-    * #### Added
     * #### Changed
         * Change the type of applicable properties and method parameters from `datetime.timedelta` to `hightime.timedelta` and from `datetime.datetime` to `hightime.datetime`. - [#744](https://github.com/ni/nimi-python/issues/744), [#1368](https://github.com/ni/nimi-python/issues/1368), [#1382](https://github.com/ni/nimi-python/issues/1382), [#1397](https://github.com/ni/nimi-python/issues/1397)
         * Update "Driver Version Tested Against", in documentation, with latest versions installed on nimi-bot. The version is 20.0.0 for all modules except nidigital, for which it is 19.0.1.
-    * #### Removed
 * ### NI-DCPower
     * #### Added
         * API parity with NI-DCPower 20.0 by adding the following properties:
             * `Session.serial_number`
             * `Session.actual_power_allocation`
             * `Session.requested_power_allocation`
             * `Session.power_allocation_mode`
-    * #### Changed
-    * #### Removed
 * ### NI-Digital Pattern Driver
```

Comment:
Something should be said about this being a 0.9.0 release that is considered API complete.

Reply:
done